etna/core/static/admin/js/inputLengthIndicators.js

```diff
@@ -0,0 +1,69 @@
+const lengthIndicatorClassname = "inputlengthindicator";
+const lengthIndicatorExceededClassname = "inputlengthindicator--exceeded";
+const countHTMLBase = '<span class="w-sr-only">Character count:</span> ';
+
+const countChars = function (text) {
+  /*
+  Count characters in a string, with special processing to account for astral symbols in UCS-2. See:
+  - https://github.com/RadLikeWhoa/Countable/blob/master/Countable.js#L29
+  - https://mathiasbynens.be/notes/javascript-unicode
+  - https://github.com/tc39/proposal-intl-segmenter
+  */
+    if (text) {
+        // Find as many matches as there are (g), matching newlines as characters (s), as unicode code points (u).
+        // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#advanced_searching_with_flags.
+        const matches = text.match(/./gsu);
+        return matches ? matches.length : 0;
+    }
+    return 0;
+};
+
+const updateLengthIndicator = function (
+    lengthIndicator,
+    length,
+    maxChars,
+) {
+    lengthIndicator.innerHTML = countHTMLBase + `${length}/${maxChars}`;
+
+    if (length > maxChars) {
```

Comment:
Thought: As we're using max-length as selector, will this max-limit ever be exceded? It's enforced client side from what I can see.

Reply:
You're right, that's unlikely. This is actually a remnant of when I was supporting a `max-words` option, which doesn't benefit from any client-side enforcement. Saying that, we have had some situations where we have changed the `max_length` value, and existing values might need to be trimmed down to bring them into line - in which case, this may still be of some benefit.